scripts/install

```diff
@@ -172,20 +208,7 @@ if [[ ! -v ip6_net ]]; then # if ip6_net not set
   fi
 fi
 
-wifi_device=$(sudo bash ../conf/iw_devices | awk -F\| '{ print $1 }')
-
-# Save arguments
-
-if [[ -z $wifi_device ]]; then
-  ynh_app_setting_set $app service_enabled 0
-  wifi_device=none
-else
-  ynh_app_setting_set $app service_enabled 1
-fi
-
-#=================================================
-# SAVE SETTINGS
-#=================================================
+wifi_device=$(bash ../conf/iw_devices | awk -F\| '{ print $1 }')
```

Comment:
This line does not work as `iw_devices` runs `iw` and `iw` requires `sudo`.
Thus the device is not detected and the hotspot does not start.

Reply:
Actually, this is launched by user `admin`, so access rights is not the issue but PATH.

````
> echo $PATH
/usr/local/bin:/usr/bin:/bin:/usr/local/games:/usr/games
```

The `iw` program is located under `/sbin/`, which is not in user `admin` PATH.
We should either but the complete path to `iw` in the script `iw_devices` (and others) or add `/sbin/` to the admin's PATH.